### ISSUE TITLE
Don't fail script for missing card type

### ISSUE THREAD

**Issue by tooomm** [Cockatrice/Magic-Spoiler#301]
Recent runs started to fail and spoilers are not updated since 2 days:
<img width="495" alt="image" src="https://github.com/Cockatrice/Magic-Spoiler/assets/9874850/1d25eb49-5b50-43cb-bc1b-bc4b09e47800">

**Comment by ebbit1q** [comment]
yes, there seems to be a card without type line causing the script to break

**Comment by tooomm** [comment]
Can we make it a warning instead?
An empty title is not that bad maybe.

If you know which card it is, you can also report it to Scryfall, they have a link at the bottom of each card:
<img width="236" alt="image" src="https://github.com/Cockatrice/Magic-Spoiler/assets/9874850/aacbc88c-142d-40f9-b337-62e84305e9b2">

**Comment by ebbit1q** [comment]
it's a preview card, it's likely the type was simply not known yet at the time

**Comment by tooomm** [comment]
Scryfall is quite good with their data, unusual that they have a picture but provide incomplete data otherwise. If we did report the faulty card, they would have resolved it within less than a day I guess.

**Anyway, the script should not simply fail here.
We are ok with having a missing type for a single card I guess. Once known it will be included with the next update.  
Right now the behaviour blocks getting any other updates.**

**Comment by tooomm** [comment]
This is happening again, we are not updating spoilers since two weeks.
